blue-xdma/src/include/XDMADescriptorGenerator.bsv

```diff
@@ -4,69 +4,65 @@ import GetPut :: *;
 import BlueAXI :: *;
 import BlueLib :: *;
 import AXI4LiteMaster :: *;
-import AXI4Slave :: *;
 import StmtFSM :: *;
 import Config :: *;
+import AXI4Slave :: *;
 
 typedef struct {
     XDMADescriptorLength length;
     XDMADescriptorAddressSz srcAddr;
     XDMADescriptorAddressSz dstAddr;
 } XDMADescriptor deriving (Bits, Eq, FShow);
 
+typedef enum {C2H, H2C} Direction deriving(Eq, Bits, FShow);
+
 typedef struct {
-    Bool isC2H;
+    Direction dir;
     Bool enable;
-} IfcControlDMA deriving (Bits, Eq, FShow);
+} DMA_Configure deriving (Bits, Eq, FShow);
```

Comment:
strict camel style: `ConfigureDMA`

Reply:
In the `bsc` project, there is also a mixed naming style of snakeCase and camelCase, such as `BRAM_Configure`.

https://github.com/B-Lang-org/bsc/blob/0eb551dcdb2fad5dd455d5937519b768ddb61fd4/src/Libraries/Base3-Misc/BRAM.bsv#L48